docs/canvas/canvas-elements.asciidoc

```diff
@@ -0,0 +1,259 @@
+
+[float]
+[[add-canvas-element]]
+=== Add elements to your workpad
+
+Choose the elements to display on your workpad, then familiarize yourself with the element using the preconfigured demo data.
+
+. Click *Add element*.
+
+. In the *Elements* window, select the element you want to use. 
++
+[role="screenshot"]
+image::images/canvas-element-select.gif[Canvas elements]
+
+. Play around with the default settings and see what the element can do. 
+
+TIP: Want to use a different element? You can delete the element by selecting it, clicking the *Element options* icon in the top right corner, then selecting *Delete*.
+
+[float]
+[[connect-element-data]]
+=== Connect the element to your data
+
+When you are ready to move on from the demo data, connect the element to your own data.
+
+. Make sure that the element is selected, then select *Data*.
+
+. Click *Change your data source*.
+
+[float]
+[[elasticsearch-sql-data-source]]
+==== Connect to Elasticsearch SQL
+
+Access your data in Elasticsearch using the Elasticsearch SQL syntax.
+
+Unfamiliar with writing Elasticsearch SQL queries? For more information, refer to {ref}/sql-spec.html[SQL language].
+
+. Click *Elasticsearch SQL*.
+
+. In the *Elasticearch SQL query* box, enter your query, then *Preview* it.
+
+. If everything looks correct, *Save* it. 
+
+[float]
+[[elasticsearch-raw-doc-data-source]]
+==== Connect to Elasticsearch raw data
+
+Use the Lucene query syntax to use your raw data in Elasticsearch.
+
+For for more information about the Lucene query string sytax, refer to <<lucene-query,Lucene Query Syntax>>.
+
+. Click *Elasticsearch raw documents*.
+
+. From the *Index* dropdown, select the index data that you want to display. 
+
+. From the *Fields* dropdown, select the associated fields you want to display.
+
+. To sort the data, select an option from the *Sort Field* and *Sort Order* dropdowns.
+
+. For more targeted queries, enter a *Query* using the Lucene query string syntax. 
+
+. *Preview* the query. 
+
+. If your query looks correct, *Save* it. 
+
+[float]
+[[timelion-data-source]]
+==== Connect to Timelion
+
+Use <<timelion,Timelion>> queries to use your time series data. 
+
+. Click *Timelion*.
+
+. Enter a *Query* using the Lucene query string syntax. 
++
+For for more information about the Lucene query string syntax, refer to <<lucene-query,Lucene Query Syntax>>.
+
+. Enter the *Interval*, then *Preview* the query.
+
+. If your query looks correct, *Save* it. 
+
+[float]
+[[configure-display-options]]
+=== Choose the display options
+
+Choose how you want the elements to appear on your workpad.
+
+[float]
+[[process-and-display]]
+==== Specify how to process and display the data
+
+When you connect your element to a data source, the element often appears as a warning. To remove the error, specify the process and display options.
+
+. Click *Display*
+
+. Change the data points to display in your element. 
+
+. In the *Label* field, enter a name for the data.
```

Comment:
Which element are these instructions for? Not every element has a `Label` option, so it might be good to specify the type of element we're referring to here and providing a screenshot of the element and the sidebar options.